src/index.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const { git, gitMultilineResults, toTitleCase } = require('./utils');
+
+let packageName = JSON.parse(fs.readFileSync('./package.json').toString()).name;
+
+const packageNameAliases = {
+	'balena.io': 'resin-api',
+};
+
+packageName = packageNameAliases[packageName] || packageName;
+
+const moduleAliases = {
+	'resin-ui': 'dashboard',
+};
+
+const moduleName =
+	moduleAliases[packageName] ||
+	packageName.replace('resin-', '').replace('balena-', '');
+
+function getLatestProdInfo() {
+	const tags = gitMultilineResults('tag -l --sort=refname production-*');
+	let latestProdTag = tags[tags.length - 1];
+
+	// if the current commit has the production tag skip it
+	const currentCommitHash = git('rev-list -n 1 HEAD');
+	const latestProdTagCommitHash = git(`rev-list -n 1 ${latestProdTag}`);
+
+	if (currentCommitHash === latestProdTagCommitHash) {
+		latestProdTag = tags[tags.length - 2];
+	}
+
+	let latestProdVersionTag = gitMultilineResults(
+		`log ${latestProdTag} --skip 0 -n 1 --pretty=tformat:%s`,
+	).filter(line => line.match(/^v/))[0];
+	if (!latestProdVersionTag) {
+		// for VB v1 tags
+		latestProdVersionTag = gitMultilineResults(
+			`log ${latestProdTag} --skip 1 -n 1 --pretty=tformat:%s`,
+		).filter(line => line.match(/^v/))[0];
+	}
+	return {
+		latestProdTag,
+		latestProdVersionTag,
+	};
+}
+
+const latestProdInfo = getLatestProdInfo();
+const prevVersionTag = latestProdInfo.latestProdVersionTag;
+
+const rawChangelog = gitMultilineResults(
+	`diff -U0 ${prevVersionTag} CHANGELOG.md`,
```

Comment:
If the component has a `.versionbot` folder, that will contain a YAML version of the changelog. VB will start filling in that file if it finds a file at `.versionbot/CHANGELOG.yml`, I guess most repos still don't have this but it would be nice to start adding it for these cases

Reply:
We can open an issue for this :)